prisma/schema.prisma

```diff
@@ -204,11 +202,29 @@ model Judgement {
 }
 
 model ReactionCount {
-  type ReactionType
-  count Int @default(0)
+  type  ReactionType
+  count Int          @default(0)
 
   project   Project @relation(fields: [projectId], references: [id])
   projectId String
 
   @@id([type, projectId])
 }
+
+model Photo {
+  id        String   @id @default(cuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  url       String
+  urlLarge  String
```

Comment:
I think urlLarge, urlMedium, and urlSmall should be optional. Any new photos uploaded can go through the Uploader service which allows us to resize images automatically on the fly. It's only old photos which would need these, and in the long term we can probably migrate them to uploader as well.